client/state/timezones/reducer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	DESERIALIZE,
+	SERIALIZE,
+
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS,
+} from 'state/action-types';
+import {
+	rawOffsetsSchema,
+	timezonesSchema
+} from './schema';
+
+const offsets = createReducer( [], {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets.slice( 0 )
+}, rawOffsetsSchema );
+
+const timezones = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { timezonesByContinent } ) => ( timezonesByContinent )
+}, timezonesSchema );
+
+export const items = combineReducers( {
```

Comment:
>I think do you meant to `state.timezones.timezones_by_contienet`? I prefer to keep them separately since they are different things.`byContinent` sounds great, though.

Currently it's:

```
state.timezones.items.rawOffsets
state.timezones.items.timezones
state.timezones.requesting
```

I'm suggesting:

```
state.timezones.rawOffsets
state.timezones.timezones // Or: `items`, `byContinent`
state.timezones.requesting
```

Per my second note, even:

```
state.timezones.rawOffsets
state.timezones.items
state.timezones.byContinent // Mapping continent to array of `item` keys
state.timezones.requesting
```

Speaking of keying, should we be transforming these into keyed objects for easier lookup, instead of leaving them as arrays?